venues/auai.org/UAI/2018/webfield/areachairWebfield.html

```diff
@@ -0,0 +1,442 @@
+<!-- UI for Area Chair Console -->
```

Comment:
I'm not sure - the ICLR area chair webfields are all html, for some reason. I remember that there was some exception that was preventing us from using js extensions, but I don't remember why.

@zbialecki , can you confirm?

Reply:
I think the ICLR area chair webfield was created before I made the new webfield module so it was originally designed to be HTML. We could try changing it to be pure JS, but I think we ran into a problem with how the current code accesses global variables. The new style js webfields can only access certain global variables. It would be worth testing again though.